### PR TITLE
Feature/quick landing page modification

### PR DESCRIPTION
**Pull request by bneiluj**

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
 			}
 		},
 		"walletConnection": {
-			"title": "Please connect a wallet with your SNX holdings to start:"
```
Comment (leopoldjoy) [inline public/locales/en/translation.json:26]: Best to have descriptive text, no? How about:

```
Please connect a wallet containing your SCT tokens to start:
```

File: src/helpers/networkHelper.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const INFURA_JSON_RPC_URLS = {
 	42: `https://kovan.infura.io/v3/${INFURA_PROJECT_ID}`,
 };
 
-export const SUPPORTED_WALLETS = ['Metamask', 'Trezor', 'Ledger', 'Coinbase', 'WalletConnect'];
```
Comment (leopoldjoy) [inline src/helpers/networkHelper.js:30]: Why? Because we don't like Coinbase?

Comment (bneiluj) [reply src/helpers/networkHelper.js:30]: When i tested it it was having issues to connect using localhost so i disable it. We can add it later and make announcement about it.

File: public/locales/en/translation.json (hunk 1 above)
Comment (bneiluj) [reply public/locales/en/translation.json:26]: Not urgent for ED but will modify it

**Pull request merged**